example/blog/tests/test_upload_image.py

```diff
@@ -19,7 +19,7 @@ def test_upload_file(admin_client, file):
 def test_upload_file_to_google_cloud(admin_client, file, settings):
```

Comment:
this test is oddly now failing for me but I think that's because it was not actually using `storages.backends.gcloud.GoogleCloudStorage`. I added some print statements and on master it uses `articles.storage.CustomStorage`, somehow `override_settings` was not working and now is. I'm not entirely sure why. Maybe we can remove this test as I personally don't want to make a request to the google cloud servers every time I run the tests. What do you think @hvlads ?